node_modules/oae-core/linkpreview/js/linkpreview.js

```diff
@@ -135,9 +137,19 @@ define(['jquery', 'oae.core', 'jquery.oembed'], function($, oae) {
                     embeddable = false;
                 }
             }
+            // Only embed text, html, images, and types that the browser has plugins to support
+            var type = widgetData.previews.targetType ? widgetData.previews.targetType.split(';')[0] : false;
+            var universalSupport = _.contains(['text/html', 'text/plain', 'image/png', 'image/jpeg', 'image/gif', 'image/svg+xml'], type);
+            var havePlugin = navigator.mimeTypes && navigator.mimeTypes[type] && navigator.mimeTypes[type].enabledPlugin;
+            var supported = widgetData.previews.targetType && (universalSupport || havePlugin);
+
             // The link can be embedded as an iFrame
-            if (embeddable) {
-                oae.api.util.template().render($('#linkpreview-iframe-template', $rootel), {'link': link}, $('#linkpreview-container'), $rootel);
+            if (embeddable && supported) {
+                if (type.substring(0, 6) === 'image/') {
+                    oae.api.util.template().render($('#linkpreview-image-template', $rootel), {'img': link, 'linkProfile': widgetData}, $('#linkpreview-container'), $rootel);
```

Comment:
I see that you've addressed the above comment here. However, I can't seem to get this to work. I added the link `http://www.sanfrancisco.travel/sites/sftraveldev.prod.acquia-sites.com/files/SanFrancisco_0.jpg`, which resulted in the following content object:

```
{
    "tenant": {
        "alias": "cam",
        "displayName": "Cambridge University"
    },
    "id": "c:cam:QJi_TtiuI",
    "visibility": "public",
    "displayName": "http://www.sanfrancisco.travel/sites/sftraveldev.prod.acquia-sites.com/files/SanFrancisco_0.jpg",
    "description": "",
    "resourceSubType": "link",
    "createdBy": {
        "tenant": {
            "alias": "cam",
            "displayName": "Cambridge University"
        },
        "id": "u:cam:m1YsO2xr",
        "displayName": "Nicolaas Matthijs",
        "visibility": "public",
        "email": "nicolaasmatthijs@googlemail.com",
        "locale": "en_GB",
        "publicAlias": "Nicolaas Matthijs",
        "picture": {},
        "profilePath": "/user/cam/m1YsO2xr",
        "resourceType": "user",
        "notificationsUnread": 4,
        "notificationsLastRead": 1422633776025,
        "acceptedTC": 0,
        "lastModified": 1425443964716,
        "emailPreference": "immediate"
    },
    "created": "1423339918177",
    "lastModified": "1423339918184",
    "profilePath": "/content/cam/QJi_TtiuI",
    "resourceType": "content",
    "latestRevisionId": "rev:cam:m1h_aKsuU",
    "previews": {
        "embeddable": true,
        "targetType": "image/jpeg",
        "httpsAccessible": false,
        "status": "done",
        "total": 4,
        "thumbnailUrl": "/api/download/signed?uri=local%3Ac%2Fcam%2FQJ%2Fi_%2FTt%2Fiu%2FQJi_TtiuI%2Fpreviews%2Frev-cam-m1h_aKsuU%2Fthumbnail.jpg&expires=1426723200000&signature=a0cfdd57e281bff33d17c8a97aa5c94a6519aee2",
        "smallUrl": "/api/download/signed?uri=local%3Ac%2Fcam%2FQJ%2Fi_%2FTt%2Fiu%2FQJi_TtiuI%2Fpreviews%2Frev-cam-m1h_aKsuU%2Fsmall.png&expires=1426723200000&signature=d89dbf532aea0bb35319f6f666541d70f79782a5",
        "mediumUrl": "/api/download/signed?uri=local%3Ac%2Fcam%2FQJ%2Fi_%2FTt%2Fiu%2FQJi_TtiuI%2Fpreviews%2Frev-cam-m1h_aKsuU%2Fmedium.png&expires=1426723200000&signature=2cf352802d45918c4ab92319de0f7e9fda17c0cb",
        "largeUrl": "/api/download/signed?uri=local%3Ac%2Fcam%2FQJ%2Fi_%2FTt%2Fiu%2FQJi_TtiuI%2Fpreviews%2Frev-cam-m1h_aKsuU%2Flarge.png&expires=1426723200000&signature=5365a88cd9164991904bef28e4c906e92f48596e"
    },
    "link": "http://www.sanfrancisco.travel/sites/sftraveldev.prod.acquia-sites.com/files/SanFrancisco_0.jpg",
    "signature": {
        "expires": 1425988800000,
        "signature": "8b372cf4ce1476cabb4b69e288d729a03fc62d8a"
    },
    "isManager": true,
    "canShare": true
}
```

This still causes the image to be embedded through an iframe though